src/Microsoft.AspNet.Authentication/AuthenticateResult.cs

```diff
@@ -44,6 +46,11 @@ public static AuthenticateResult Success(AuthenticationTicket ticket)
             return new AuthenticateResult() { Ticket = ticket };
         }
 
+        public static AuthenticateResult Skip()
+        {
+            return new AuthenticateResult() { Skipped = true };
+        }
+
         public static AuthenticateResult Failed(Exception error)
```

Comment:
Also, having a parameterless `Fail`/`Failed` overload would be really useful: https://github.com/aspnet/Security/pull/568#issuecomment-162691988